akka-http-core/src/main/mima-filters/10.1.9.backwards.excludes

```diff
@@ -0,0 +1,4 @@
+# https://github.com/akka/akka-http/issues/2605
```

Comment:
Let's start creating excludes files per issue to avoid conflicts.

So, it should go into 

```
mima-filters/10.1.9.backwards.excludes/issue-2605-deprecate-right-method.excludes
```

(or similar).


See https://github.com/lightbend/mima/commit/2573ceeb4a222efb8405ab0793b33e3ba930ca60#diff-eec91a6d045cc72eff0f61fed09b4ac3 for more info about that feature.